commons-core/src/main/scala/com/avsystem/commons/SharedExtensions.scala

```diff
@@ -253,7 +255,7 @@ object SharedExtensions extends SharedExtensions {
       fut.transform(s, f)(RunNowEC)
 
     def transformNow[S](f: Try[A] => Try[S]): Future[S] =
-      fut.transformTry(f)(RunNowEC)
+      fut.transform(f)(RunNowEC)
```

Comment:
Any thoughts on `RunNowEC` vs `scala.concurrent.ExecutionContext.parasitic` in 2.13?

Reply:
We can think about it when we drop 2.12.